apps/closures-and-restrictions-map/src/components/Layers.tsx

```diff
@@ -45,10 +43,24 @@ export const Layers = ({ filter }: ILayerProps) => {
     // },
   ];
 
+  const layerHandler = (layer: string) => {
+    const activeStatusKeys = filter.activeKeys;
+    if (activeStatusKeys.length === 1 && activeStatusKeys.includes(layer)) return;
```

Comment:
would prefer:

```suggestion
    const activeStatusKeys = filter.activeKeys;
    if (isDisabled(layer)) return;
```

together with moving `isDisabled` above the `layerHandler`

so that it does not desync in the future